apps/openmw/mwbase/mechanicsmanager.hpp

```diff
@@ -97,7 +97,7 @@ namespace MWBase
             virtual int getBarterOffer(const MWWorld::Ptr& ptr,int basePrice, bool buying) = 0;
             ///< This is used by every service to determine the price of objects given the trading skills of the player and NPC.
 
-            virtual int getDerivedDisposition(const MWWorld::Ptr& ptr) = 0;
+            virtual int getDerivedDisposition(const MWWorld::Ptr& ptr, bool addTemporaryDispositionChange = false) = 0;
```

Comment:
I  think it would be better for the parameter default to be ```true```, since that's what most callers use. The only exception where we need ```false``` is when the temporary disposition change is recalculated in DialogueManager::persuade.